### PR TITLE
Added releaser documentation.

### PR DESCRIPTION
**Pull request by Aaronius**
#### Changes
Added details about the Releaser tool. Replaced the old-school cURL way.

### CODE REVIEW

File: extensions/submissions/release.md [hunk 1]
```diff
@@ -16,50 +16,12 @@ You must release your extension as a private extension before you can release it
 
 ## Release - Private
 
-You release your extension package as private by `PATCH`ing your `extension_package` resource and supplying a payload with an instruction to make the `extension_package` private.
+The easiest way to release your extension to private availability is to use the [Launch Extension Releaser](https://www.npmjs.com/package/@adobe/reactor-releaser). More instructions are found within its documentation.
```
Comment (thebenrob) [inline extensions/submissions/release.md:19]: This guide is supposed to be a step-by-step thing.  I think we should still walk them through the steps to using it here.

Comment (Aaronius) [reply extensions/submissions/release.md:19]: Are you talking about basically copying what's on the linked page (https://www.npmjs.com/package/@adobe/reactor-releaser)?

Comment (thebenrob) [reply extensions/submissions/release.md:19]: Higher level than what's in NPM.  More a conversational walkthrough of what the CLI is going to ask and what you should put in.  Essentially matching what is done on https://developer.adobelaunch.com/extensions/submissions/upload-and-test/#3-upload-your-extension-package for how to use the Uploader tool.

Comment (Aaronius) [reply extensions/submissions/release.md:19]: Okay, I won't have time to address this comment for a while. This documentation needs updated as well: https://developer.adobelaunch.com/extensions/submissions/upload-and-test/#3-upload-your-extension-package

Comment (thebenrob) [reply extensions/submissions/release.md:19]: If you don't have time to do it now, I don't think we should hold up the release for it.  I have an item on my docs backlog to address this eventually, so maybe we let this go as is, and we can come back and fix it later.

**Review by thebenrob — APPROVED** [review]

**Pull request merged**